### PR TITLE
fix: use input_field_generator inside the group control for consistent input generation

### PR DESCRIPTION
**Pull request by FlutterJoey**

### CODE REVIEW

**Review by JacquesDIconica — APPROVED** [review]

File: lib/src/ui/input_field_generator.dart [hunk 1]
```diff
@@ -56,11 +56,13 @@ class _InputFieldGeneratorState extends State<InputFieldGenerator> {
                   var isLastSetting = i == setting.settings!.length - 1;
                   setting.settings![i].isLastSettingInGroup = isLastSetting;
                   setting.settings![i].partOfGroup = true;
-                  var currentSetting = setting.settings![i];
 
-                  return _buildInputFieldFromControlSetting(
```
Comment (Gorter-dev) [inline lib/src/ui/input_field_generator.dart:61]: Assuming this was recursive for pages in pages. Why is this removed?

Comment (FlutterJoey) [reply lib/src/ui/input_field_generator.dart:61]: It is still recursive, except it uses the class rather than the method.

**Review by Gorter-dev — APPROVED** [review]
